scripts/pack/pack.js

```diff
@@ -0,0 +1,26 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
Which dependency is throwing the error for being a dev-dependency? Any dependency used here should be shipped for others to use this script

Reply:
Shelljs, I'll update this to be a dependency. I don't think it'll cause any issues.

Reply:
Moved to dependency: https://github.com/cerner/terra-toolkit/pull/198/commits/2481a6e49cead4eba3e9bbdec6344d1009f41be6